draft-ietf-shmoo-hackathon.md

```diff
@@ -479,7 +479,7 @@ The email alias, {{EMAIL-ALIAS}}, was created and is maintained by the Secretari
 
 The {{IETF-HACKATHON-GITHUB}} is used to share code, presentations, and other artifacts at IETF Hackathons. The Hackathon Chairs are responsible for administering the GitHub organization.
 
-Code for Hackathon projects often exist elsewhere, which is perfectly fine. Anyone needing a place to host code for the Hackathon can request the creating of a repository for their project.
+Code for Hackathon projects often exist elsewhere, which is perfectly fine. Anyone needing a place to host code for the Hackathon can request the creation of a repository for their project in addition to sharing their presentation (see 4.3 Upload to Github).
```

Comment:
Good catch on s/creating/creation. I think it best to omit mention of the presentations here to avoid confusion as to where the presentations should be uploaded.